.drone.star

```diff
@@ -58,6 +58,13 @@ dirs = {
     "ocmProviders": "tests/config/drone/providers.json",
 }
 
+# OCIS URLs
+OCIS_SERVER_NAME = "ocis-server"
```

Comment:
If ocis server name is `ocis-server`, why use a variable and not use the string directly?
If this is changed somehow, will `OCIS_FED_URL` always be `https://federation-ocis-server:10200`?

Reply:
`OCIS_FED_URL` is independent of ocis server naming.

Reply:
>If ocis server name is ocis-server, why use a variable and not use the string directly?

we can but this is to maintain single source of truth. if we change ocis service pipeline step name to something else then we have to make changes to all the lines where the ocis url is being used.
So, doing this, we can make a change in this variable then everything will work just fine.